playbooks/nuke.yml

```diff
@@ -0,0 +1,241 @@
+          - "{{ (config_base_path, 'confluent-metadata-service') | path_join }}"
+          - "{{ (config_base_path, 'confluent-rebalancer') | path_join }}"
+
+    - name: Find Installed Confluent Service files
+      set_fact:
+        cp_services: "{{ confluent_services.files | map(attribute='path') | map('basename') }}"
+
+    - name: Set common extention names in ssl files
+      set_fact:
+        ssl_extentions: [ ".crt", ".key", ".chain", ".keystore.jks", ".truststore.jks", ".keystore.bcfks", ".truststore.bcfks" ]
+
+    - set_fact:
+        ssl_items: "{{ ssl_common_items + [item] | product(ssl_extentions) | map('join') | list }}"
+        cp_packages: "{{ cp_packages | default([]) + [ lookup('vars', item + '_packages', default='')  ]  }}"
+        config_dirs: "{{ config_dirs | default(cp_default_config_dirs) + [ lookup('vars', item + '_config_prefix_path', default='') ] }}"
+        logs_dir: "{{ logs_dir | default([]) + [ lookup('vars', item + '_log_dir', default='') ] }}"
+        cp_users: "{{ cp_users|default(cp_default_users) + [ lookup('vars', item+ '_user', default='') ] }}"
+        cp_groups: "{{ cp_groups|default([]) + [ lookup('vars', item+ '_group', default='') ] }}"
+        keytabs_filenames: "{{ keytabs_filenames|default([])  + [ ('/etc/security/keytabs/', item, '.keytab') | join ] }}"
+        cp_services: "{{ cp_services|default([]) + [ lookup('vars', item+'_service_name', default='') ] }}"
+        secrets_protection_files: "{{ secrets_protection_files|default([kafka_broker_client_secrets_protection_file]) + [ lookup('vars', item+'_secrets_protection_file', default=(item|replace('_','-')+'-secrets-protection.properties')) ] }}"
+      with_items: "{{ group_names }}"
+
+    - set_fact:
+        data_dirs: "{{ data_dirs + [ control_center_rocksdb_path|default('') ] }}"
+      when: inventory_hostname in groups.control_center
+
+    - set_fact:
+        data_dirs: "{{ data_dirs + [ ksql_rocksdb_path| default('/tmp/ksqldb'), ksql_final_properties['ksql.streams.state.dir'] ] }}"
+      when: inventory_hostname in groups.ksql
+
+    - name: List Package names to be deleted
+      debug:
+        msg: "{{ cp_packages | flatten| unique }}"
+    - name: List Keytab file names to be deleted
+      debug:
+        msg: "{{ keytabs_filenames }}"
+    - name: List Linux Groups to be deleted
+      debug:
+        msg: "{{ cp_groups | select() }}"
+    - name: List Linux Users to be deleted
+      debug:
+        msg: "{{ cp_users  | select() }}"
+    - name: List Services to be disabled and deleted
+      debug:
+        msg: "{{ cp_services | select() }}"
+    - name: List Data and config Directories to be deleted
+      debug:
+        msg:
+          - "{{ logs_dir | select() }}"
+          - "{{ ssl_items | select() }}"
+          - "{{ secrets_protection_files | select() }}"
+          - "{{ config_dirs | select() }}"
+          - "{{ data_dirs | select() }}"
+
+    - name: Re-confirm
```

Comment:
Lets add a flag to override the prompt. The default behaviour should be to prompt the user